lblod/frontend-gelinkt-notuleren
# Additional LMB IV GR corrections

### Overview
This PR adds some additional corrections to the mandatee table configuration for the 'gemeenteraadsinstallatievergadering'.
Specifically, it includes:
- Removal of `mandaat:bekrachtigdAanstellingVan` where not applicable
- Some adjustments in how the start and end date of a mandate is determined
- An additional entry `IVGR7-LMB-3-verhindering-schepenen` which includes a list of schepenen with status 'verhinderd'.

##### connected issues and PRs:
None

### Setup
None

### How to test/reproduce
- Start the application and open/create an IV
- Ensure that the following tables no longer contain the non-applicable 'mandaat:bekrachtigdAanstellingVan` relationship
   * `IVGR7-LMB-1-kandidaat-schepenen`
   * `IVGR8-LMB-1-verkozen-schepenen`
- Ensure that the following table now contains the start (and optionally the end) of the mandates as listed on LMB (and no longer the date of the meeting)
   * `IVGR7-LMB-2-ontvankelijkheid-schepenen`
- Test out the new `IVGR7-LMB-3-verhindering-schepenen`, it should only contain 'schepenen' from LMB who are 'verhinderd'



### Checks PR readiness
